### ISSUE TITLE
Question about the dataset and how to insert a picture

### ISSUE THREAD

**Issue by boyaow** [DS4PS/ddmp-uw-class-spring-2019#2]
1.
![屏幕快照 2019-04-03 下午2 21 48](https://user-images.githubusercontent.com/47672261/55519230-cfb77500-562b-11e9-9e21-6bde667cf9b6.png)
Why is some datasets in the category of "Data" and others in the category of "Value"? What does "Promise"mean?
2.
![屏幕快照 2019-04-03 下午2 36 53](https://user-images.githubusercontent.com/47672261/55519303-1ad18800-562c-11e9-94b3-dc93ca0e3937.png)

![屏幕快照 2019-04-03 下午4 20 03](https://user-images.githubusercontent.com/47672261/55519368-5f5d2380-562c-11e9-9c18-c3c5d90c50e3.png)
I try to insert the picture in the markdown but an error still shows up.  Looking back at the code, there seems no problem because the picture appear right under the code. What is the problem?
Thank you.

**Comment by lecy** [comment]
Promise means the dataset exists in R, but has not yet been loaded. If you type for example:

```r
summary( crimtab )
```

You will see the "promise" turns into the meta-data you see for others. 

-----

Regarding your picture, I think the tilde is confusing R. When you knit, R will always look for the file in the same directory as the RMD file. If it's in a folder on the desktop, for example, you would write:

```
![](images/me.png)
```

The tilde ~ tells the program to start from the base directory (usually C:). The real-time preview might understand that convention, but the knitr package does not, which is why you get the error.

**Comment by boyaow** [comment]
Thank you for your help and instructions. I tried the code `![](images/me.png)`but it still shows no path for the image, is there anything wrong? I am confused.
![屏幕快照 2019-04-03 下午7 33 27](https://user-images.githubusercontent.com/47672261/55525905-ce944100-5647-11e9-9bae-ca1947496a53.png)

**Comment by lecy** [comment]
Where is your RMD file and image file? Is the images folder inside the same folder as your RMD file?

RMD
images/me.png

**Comment by boyaow** [comment]
My RMD file and image file are in different folders so do I have tp put the images folder in the same folder as my RMD file? Thank you.

**Comment by ejvanholm** [comment]
Right, I should have clarified in class that RMD files have local awareness. So they look for the files you link to in the same folder. The easiest way to move forward is to copy whatever picture you want to use into the same folder your Rmarkdown is saving to.

The other option that is straightforward is to use a picture on the internet, you can just copy the weblink for that image using the same structure ![]().

**Comment by boyaow** [comment]
I think i understand.
